examples/with-chakra-ui-typescript/src/pages/_app.tsx

```diff
@@ -0,0 +1,17 @@
+import { ThemeProvider, CSSReset, ColorModeProvider } from '@chakra-ui/core'
+
+import theme from '../theme'
+import { AppProps } from 'next/app'
+
+function MyApp({ Component, pageProps }: AppProps) {
+  return (
+    <ThemeProvider theme={theme}>
+      <ColorModeProvider>
```

Comment:
Chakra v1 is [very close](https://next.chakra-ui.com/docs/migration). Would it make more sense to upgrade to that RC here because it has improved color mode support?

I'd hate to have people start with this example and then try to update to latest Chakra and get bit with breaking changes.

Reply:
Nice.. I will update it on the weekend!